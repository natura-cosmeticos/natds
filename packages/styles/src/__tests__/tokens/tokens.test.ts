import { tokens } from '../../index'

describe('Tokens', () => {
  test('It should match tokens snapshot', () => {
    expect(tokens).toMatchSnapshot('Tokens Snapshot')
  })
})
