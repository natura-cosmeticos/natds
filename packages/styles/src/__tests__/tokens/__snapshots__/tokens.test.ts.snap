// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tokens It should match tokens snapshot: Tokens Snapshot 1`] = `
Object {
  "avatarSizes": Object {
    "huge": Object {
      "fontSize": "1.5rem",
      "size": 80,
    },
    "large": Object {
      "fontSize": "1rem",
      "size": 48,
    },
    "small": Object {
      "fontSize": ".875rem",
      "size": 32,
    },
    "standard": Object {
      "fontSize": "1rem",
      "size": 40,
    },
    "tiny": Object {
      "fontSize": ".625rem",
      "size": 24,
    },
  },
  "buttonSizes": Object {
    "large": Object {
      "height": 48,
      "paddingBottom": 0,
      "paddingLeft": 16,
      "paddingRight": 16,
      "paddingTop": 0,
    },
    "medium": Object {
      "height": 40,
      "paddingBottom": 0,
      "paddingLeft": 12,
      "paddingRight": 12,
      "paddingTop": 0,
    },
    "small": Object {
      "height": 32,
      "paddingBottom": 0,
      "paddingLeft": 8,
      "paddingRight": 8,
      "paddingTop": 0,
    },
  },
  "colorTokens": Object {
    "avonDarkColorTokens": Object {
      "colorAlert": "#e74627",
      "colorBackground": "#121212",
      "colorHighEmphasis": "#fafafa",
      "colorHighlight": "#ffffff",
      "colorLink": "#227bbd",
      "colorLowEmphasis": "#777777",
      "colorMediumEmphasis": "#bbbbbb",
      "colorOnAlert": "#ffffff",
      "colorOnBackground": "#ffffff",
      "colorOnLink": "#ffffff",
      "colorOnPrimary": "#000000",
      "colorOnPrimaryDark": "#000000",
      "colorOnPrimaryLight": "#000000",
      "colorOnSecondary": "#333333",
      "colorOnSecondaryDark": "#333333",
      "colorOnSecondaryLight": "#333333",
      "colorOnSuccess": "#ffffff",
      "colorOnSurface": "#ffffff",
      "colorOnWarning": "#333333",
      "colorPrimary": "#f091c9",
      "colorPrimaryDark": "#bc6198",
      "colorPrimaryLight": "#ffc3fc",
      "colorSecondary": "#e9e9e9",
      "colorSecondaryDark": "#b7b7b7",
      "colorSecondaryLight": "#ffffff",
      "colorSuccess": "#569a32",
      "colorSurface": "#333333",
      "colorWarning": "#fcc433",
    },
    "avonLightColorTokens": Object {
      "colorAlert": "#e74627",
      "colorBackground": "#fafafa",
      "colorHighEmphasis": "#333333",
      "colorHighlight": "#000000",
      "colorLink": "#227bbd",
      "colorLowEmphasis": "#bbbbbb",
      "colorMediumEmphasis": "#777777",
      "colorOnAlert": "#ffffff",
      "colorOnBackground": "#333333",
      "colorOnLink": "#ffffff",
      "colorOnPrimary": "#ffffff",
      "colorOnPrimaryDark": "#ffffff",
      "colorOnPrimaryLight": "#000000",
      "colorOnSecondary": "#333333",
      "colorOnSecondaryDark": "#333333",
      "colorOnSecondaryLight": "#333333",
      "colorOnSuccess": "#ffffff",
      "colorOnSurface": "#333333",
      "colorOnWarning": "#333333",
      "colorPrimary": "#de0085",
      "colorPrimaryDark": "#a70058",
      "colorPrimaryLight": "#ff56b5",
      "colorSecondary": "#e9e9e9",
      "colorSecondaryDark": "#b7b7b7",
      "colorSecondaryLight": "#ffffff",
      "colorSuccess": "#569a32",
      "colorSurface": "#ffffff",
      "colorWarning": "#fcc433",
    },
    "naturaDarkColorTokens": Object {
      "colorAlert": "#e74627",
      "colorBackground": "#121212",
      "colorHighEmphasis": "#fafafa",
      "colorHighlight": "#ffffff",
      "colorLink": "#227bbd",
      "colorLowEmphasis": "#777777",
      "colorMediumEmphasis": "#bbbbbb",
      "colorOnAlert": "#ffffff",
      "colorOnBackground": "#ffffff",
      "colorOnLink": "#ffffff",
      "colorOnPrimary": "#333333",
      "colorOnPrimaryDark": "#333333",
      "colorOnPrimaryLight": "#333333",
      "colorOnSecondary": "#000000",
      "colorOnSecondaryDark": "#000000",
      "colorOnSecondaryLight": "#000000",
      "colorOnSuccess": "#ffffff",
      "colorOnSurface": "#ffffff",
      "colorOnWarning": "#333333",
      "colorPrimary": "#f4ab34",
      "colorPrimaryDark": "#ef8426",
      "colorPrimaryLight": "#fefde8",
      "colorSecondary": "#ff6b0b",
      "colorSecondaryDark": "#ff5808",
      "colorSecondaryLight": "#fff8e1",
      "colorSuccess": "#569a32",
      "colorSurface": "#333333",
      "colorWarning": "#fcc433",
    },
    "naturaLightColorTokens": Object {
      "colorAlert": "#e74627",
      "colorBackground": "#fafafa",
      "colorHighEmphasis": "#333333",
      "colorHighlight": "#000000",
      "colorLink": "#227bbd",
      "colorLowEmphasis": "#bbbbbb",
      "colorMediumEmphasis": "#777777",
      "colorOnAlert": "#ffffff",
      "colorOnBackground": "#333333",
      "colorOnLink": "#ffffff",
      "colorOnPrimary": "#333333",
      "colorOnPrimaryDark": "#333333",
      "colorOnPrimaryLight": "#333333",
      "colorOnSecondary": "#000000",
      "colorOnSecondaryDark": "#000000",
      "colorOnSecondaryLight": "#000000",
      "colorOnSuccess": "#ffffff",
      "colorOnSurface": "#333333",
      "colorOnWarning": "#333333",
      "colorPrimary": "#f4ab34",
      "colorPrimaryDark": "#ef8426",
      "colorPrimaryLight": "#fefde8",
      "colorSecondary": "#ff6b0b",
      "colorSecondaryDark": "#ff5808",
      "colorSecondaryLight": "#fff8e1",
      "colorSuccess": "#569a32",
      "colorSurface": "#ffffff",
      "colorWarning": "#fcc433",
    },
    "theBodyShopDarkColorTokens": Object {
      "colorAlert": "#e74627",
      "colorBackground": "#121212",
      "colorHighEmphasis": "#fafafa",
      "colorHighlight": "#ffffff",
      "colorLink": "#227bbd",
      "colorLowEmphasis": "#777777",
      "colorMediumEmphasis": "#bbbbbb",
      "colorOnAlert": "#ffffff",
      "colorOnBackground": "#ffffff",
      "colorOnLink": "#ffffff",
      "colorOnPrimary": "#000000",
      "colorOnPrimaryDark": "#000000",
      "colorOnPrimaryLight": "#000000",
      "colorOnSecondary": "#000000",
      "colorOnSecondaryDark": "#000000",
      "colorOnSecondaryLight": "#000000",
      "colorOnSuccess": "#ffffff",
      "colorOnSurface": "#ffffff",
      "colorOnWarning": "#333333",
      "colorPrimary": "#62b3ae",
      "colorPrimaryDark": "#2f837f",
      "colorPrimaryLight": "#94e5e0",
      "colorSecondary": "#d2ad93",
      "colorSecondaryDark": "#a07e65",
      "colorSecondaryLight": "#ffdfc4",
      "colorSuccess": "#569a32",
      "colorSurface": "#333333",
      "colorWarning": "#fcc433",
    },
    "theBodyShopLightColorTokens": Object {
      "colorAlert": "#e74627",
      "colorBackground": "#fafafa",
      "colorHighEmphasis": "#333333",
      "colorHighlight": "#000000",
      "colorLink": "#227bbd",
      "colorLowEmphasis": "#bbbbbb",
      "colorMediumEmphasis": "#777777",
      "colorOnAlert": "#ffffff",
      "colorOnBackground": "#333333",
      "colorOnLink": "#ffffff",
      "colorOnPrimary": "#ffffff",
      "colorOnPrimaryDark": "#ffffff",
      "colorOnPrimaryLight": "#ffffff",
      "colorOnSecondary": "#ffffff",
      "colorOnSecondaryDark": "#ffffff",
      "colorOnSecondaryLight": "#ffffff",
      "colorOnSuccess": "#ffffff",
      "colorOnSurface": "#333333",
      "colorOnWarning": "#333333",
      "colorPrimary": "#004236",
      "colorPrimaryDark": "#001d10",
      "colorPrimaryLight": "#356d60",
      "colorSecondary": "#a55f53",
      "colorSecondaryDark": "#73342a",
      "colorSecondaryLight": "#d98d7f",
      "colorSuccess": "#569a32",
      "colorSurface": "#ffffff",
      "colorWarning": "#fcc433",
    },
  },
  "colors": Object {
    "natura": Object {
      "complementary": Object {
        "colorBrdNatCpBlue1": "#227bbd",
        "colorBrdNatCpBlue2": "#6BB1E2",
        "colorBrdNatCpBlue3": "#54BECF",
        "colorBrdNatCpBlue4": "#8E96A9",
        "colorBrdNatCpBrown1": "#371E14",
        "colorBrdNatCpBrown2": "#5D3B2A",
        "colorBrdNatCpBrown3": "#7A573C",
        "colorBrdNatCpGreen1": "#569a32",
        "colorBrdNatCpGreen2": "#8EBE2F",
        "colorBrdNatCpGreen3": "#C2CD00",
        "colorBrdNatCpGreen4": "#BEC950",
        "colorBrdNatCpOrange1": "#E8530E",
        "colorBrdNatCpOrange2": "#EE7D00",
        "colorBrdNatCpOrange3": "#F49B00",
        "colorBrdNatCpOrange4": "#fcc433",
        "colorBrdNatCpRed1": "#DC0719",
        "colorBrdNatCpRed2": "#e74627",
        "colorBrdNatCpRed3": "#ED743E",
        "colorBrdNatCpRed4": "#F29669",
        "colorBrdNatCpYellow1": "#CCB619",
        "colorBrdNatCpYellow2": "#E7CE22",
        "colorBrdNatCpYellow3": "#FFD600",
        "colorBrdNatCpYellow4": "#8E96A9",
      },
      "gradient": Object {
        "colorBrdNatGrad1": "linear-gradient(112deg, #FFDA1B 0%, #ff6b0b 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad10": "linear-gradient(112deg, #F17F4D 0%, #A7D8DB 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad11": "linear-gradient(112deg, #E0D5BE 0%, #F17F4D 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad12": "linear-gradient(112deg, #FEED92 0%, #F17F4D 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad13": "linear-gradient(112deg, #BDD78F 0%, #DCEFF3 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad14": "linear-gradient(112deg, #E8E0D1 0%, #DBE7B1 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad15": "linear-gradient(112deg, #FAC6AB 0%, #DCEFF3 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad16": "linear-gradient(112deg, #E8E0D1 0%, #FAC6AB 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad17": "linear-gradient(112deg, #FFEDB5 0%, #FAC6AB 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad2": "linear-gradient(112deg, #86DDD4 0%, #6F38B5 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad3": "linear-gradient(112deg, #728C9E 0%, #FD6C20 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad4": "linear-gradient(112deg, #728C9E 0%, #FEE23E 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad5": "linear-gradient(112deg, #86DDD4 0%, #FD6C20 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad6": "linear-gradient(112deg, #CFDD42 0%, #86DDD4 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad7": "linear-gradient(112deg, #A7D8DB 0%, #755847 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad8": "linear-gradient(112deg, #E0D5BE 0%, #BDD78F 100%) 0% 0% no-repeat padding-box",
        "colorBrdNatGrad9": "linear-gradient(112deg, #F17F4D 0%, #BDD78F 100%) 0% 0% no-repeat padding-box",
      },
      "grayscale": Object {
        "colorBrdBlack": "#333333",
        "colorBrdNatGray": "#333333",
        "colorBrdNatGray1": "#777777",
        "colorBrdNatGray2": "#bbbbbb",
        "colorBrdNatGray3": "#EEEEEE",
      },
      "level": Object {
        "colorLevelBronze": "#C47F5B",
        "colorLevelDiamond": "#AEC2CE",
        "colorLevelGold": "#D8B75E",
        "colorLevelSeed": "#7B6A58",
        "colorLevelSilver": "#9D9D9C",
      },
      "primary": Object {
        "colorBrdNatPrimaryBrown": "#513528",
        "colorBrdNatPrimaryOrange": "#ff6b0b",
        "colorBrdNatPrimaryWhite": "#ffffff",
        "colorBrdNatPrimaryYellow": "#f4ab34",
      },
      "secondary": Object {
        "colorBrdNatSecondaryBlue": "#71CACD",
        "colorBrdNatSecondaryGreen": "#B3CA36",
        "colorBrdNatSecondaryPurple": "#7080A0",
        "colorBrdNatSecondaryYellow": "#FFBA1B",
      },
    },
  },
  "elevation": Object {
    "0": "none",
    "1": "0 1px 1px 0 rgba(0, 0, 0, 0.14)",
    "12": "0 12px 17px 0 rgba(0, 0, 0, 0.14)",
    "16": "0 16px 24px 0 rgba(0, 0, 0, 0.14)",
    "2": "0 2px 2px 0 rgba(0, 0, 0, 0.14)",
    "24": "0 24px 38px 0 rgba(0, 0, 0, 0.14)",
    "3": "0 3px 4px 0 rgba(0, 0, 0, 0.14)",
    "4": "0 4px 5px 0 rgba(0, 0, 0, 0.14)",
    "6": "0 6px 10px 0 rgba(0, 0, 0, 0.14)",
    "8": "0 8px 10px 0 rgba(0, 0, 0, 0.14)",
    "9": "0 9px 12px 0 rgba(0, 0, 0, 0.14)",
    "huge": "0 12px 17px 0 rgba(0, 0, 0, 0.14)",
    "hugeX": "0 16px 24px 0 rgba(0, 0, 0, 0.14)",
    "hugeXX": "0 24px 38px 0 rgba(0, 0, 0, 0.14)",
    "large": "0 6px 10px 0 rgba(0, 0, 0, 0.14)",
    "largeX": "0 8px 10px 0 rgba(0, 0, 0, 0.14)",
    "largeXX": "0 9px 12px 0 rgba(0, 0, 0, 0.14)",
    "medium": "0 4px 5px 0 rgba(0, 0, 0, 0.14)",
    "micro": "0 1px 1px 0 rgba(0, 0, 0, 0.14)",
    "none": "none",
    "small": "0 3px 4px 0 rgba(0, 0, 0, 0.14)",
    "tiny": "0 2px 2px 0 rgba(0, 0, 0, 0.14)",
  },
  "fontSize": Object {
    "body1": Object {
      "fontSize": 16,
      "fontWeight": 400,
      "letterSpacing": 0.51,
    },
    "body2": Object {
      "fontSize": 14,
      "fontWeight": 400,
      "letterSpacing": 0.22,
    },
    "button": Object {
      "fontSize": 14,
      "fontWeight": 500,
      "letterSpacing": 1.23,
      "lineHeight": 1.5,
    },
    "caption": Object {
      "fontSize": 12,
      "fontWeight": 400,
      "letterSpacing": 0.38,
    },
    "h1": Object {
      "fontSize": 96,
      "fontWeight": 400,
      "letterSpacing": 0,
    },
    "h2": Object {
      "fontSize": 60,
      "fontWeight": 400,
      "letterSpacing": 0,
    },
    "h3": Object {
      "fontSize": 48,
      "fontWeight": 400,
      "letterSpacing": 0,
    },
    "h4": Object {
      "fontSize": 34,
      "fontWeight": 400,
      "letterSpacing": 0.27,
    },
    "h5": Object {
      "fontSize": 24,
      "fontWeight": 400,
      "letterSpacing": 0,
    },
    "h6": Object {
      "fontSize": 20,
      "fontWeight": 500,
      "letterSpacing": 0.24,
    },
    "overline": Object {
      "fontSize": 12,
      "fontWeight": 500,
      "letterSpacing": 1.92,
    },
    "subtitle1": Object {
      "fontSize": 16,
      "fontWeight": 500,
      "letterSpacing": 0.13,
    },
    "subtitle2": Object {
      "fontSize": 14,
      "fontWeight": 500,
      "letterSpacing": 0.11,
    },
  },
  "iconSizes": Object {
    "huge": 144,
    "hugeX": 192,
    "hugeXX": 256,
    "hugex": 192,
    "hugexx": 256,
    "large": 72,
    "largeX": 96,
    "largeXX": 128,
    "largex": 96,
    "largexx": 128,
    "medium": 64,
    "micro": 16,
    "small": 32,
    "standard": 48,
    "tiny": 24,
  },
  "opacity": Object {
    "00": 0,
    "04": 0.04,
    "08": 0.08,
    "12": 0.12,
    "16": 0.16,
    "24": 0.24,
    "32": 0.32,
    "48": 0.48,
    "56": 0.56,
    "64": 0.64,
    "80": 0.8,
    "disabled": 0.32,
    "disabledLow": 0.24,
    "high": 0.64,
    "low": 0.12,
    "lower": 0.04,
    "medium": 0.48,
    "mediumHigh": 0.56,
    "mediumLow": 0.16,
    "opaque": 1,
    "transparent": 0,
    "veryHigh": 0.8,
    "veryLow": 0.08,
  },
  "overlay": Object {
    "solidBlack": 0.56,
  },
  "radius": Object {
    "circle": "50%",
    "large": 8,
    "medium": 4,
    "none": 0,
    "small": 2,
  },
  "sizes": Object {
    "huge": 96,
    "hugeX": 128,
    "hugeXX": 144,
    "hugeXXX": 192,
    "hugex": 128,
    "hugexx": 144,
    "hugexxx": 192,
    "large": 64,
    "largeX": 72,
    "largeXX": 80,
    "largeXXX": 88,
    "largex": 72,
    "largexx": 80,
    "largexxx": 88,
    "medium": 48,
    "mediumX": 56,
    "mediumx": 56,
    "micro": 4,
    "none": 0,
    "semi": 32,
    "semiX": 40,
    "semix": 40,
    "small": 16,
    "standard": 24,
    "tiny": 8,
    "veryHuge": 256,
    "veryhuge": 256,
  },
  "spacing": Object {
    "spacing": 8,
    "spacingHuge": 64,
    "spacingHugeX": 128,
    "spacingHugeXX": 144,
    "spacingHugeXXX": 192,
    "spacingLarge": 48,
    "spacingMicro": 4,
    "spacingNone": 0,
    "spacingSemi": 32,
    "spacingSmall": 16,
    "spacingStandard": 24,
    "spacingTiny": 8,
    "spacingVeryHuge": 256,
  },
}
`;
